roles/ci_gen_kustomize_values/tasks/generate_snippets.yml

```diff
@@ -54,7 +54,6 @@
 
 - name: Copy original values.yaml
   ansible.builtin.copy:
-    force: false  # Do not override in case of second run
```

Comment:
I'd like to get  `backup: true` then if you don't mind? That would allow to keep a nice track of the modifications to that file - it's also enabled for the other snippets.

Reply:
Thank you @cjeanner for the suggestion. Yes, I agree it is needed. Let me fix it.